.github/CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Reporting issues
 
-Please [open an issue](../../../issues/new/choose) if you find a bug or have a feature request.
+Please [open an issue](/../../../issues/new/choose) if you find a bug or have a feature request.
```

Comment:
For me it does not work [here](https://github.com/serokell/crossref-verifier/blob/2a27f239c6c7786081efbce79605bf87a1722111/.github/CONTRIBUTING.md). Why do you want to make it repository-independent? It seems to be fragile. If it's not documented anywhere (which is the case AFAIU), I think it can be silently broken anytime. So I'd rather use full link (https://github.com/serokell/crossref-verifier/issues/new/choose) instead.

Reply:
And btw, if the repo is moved somewhere outside GitHub, it will likely stop working. And if it's moved to another place on GitHub, it will normally keep working because there will be a redirection. And maybe there are ways to open this file so that the link won't work.

Reply:
Okay, since it does not work everywhere, let's use total link.

_I'm already anticipating the moment when we rename the repository and I regret it._

Reply:
Well, after renaming nothing will break because redirection (it's even if we forget to update it right after the renaming). It will just look a bit ugly (but `../../issues` doesn't look cool either, at least to me) and maybe will break after a long amount of time (most likely we'll change it earlier).